src/glv.rs

```diff
@@ -25,17 +25,23 @@ pub fn decompose_slow(scalar: &G1ScalarField) -> (G1ScalarField, G1ScalarField)
         remainder -= LAMBDA;
         quotient += 1u32;
     }
-    (G1ScalarField::from(quotient), G1ScalarField::from(remainder))
+    (
+        G1ScalarField::from(quotient),
+        G1ScalarField::from(remainder),
+    )
 }
 
 const LMDA1: u128 = 0xac45a4010001a402; // lambda high 64 bit
 const LMDA0: u128 = 0x00000000ffffffff; // lambda low 64 bit
-// const INV2: u128 = 0x1;                // 2**256 // lambda - 128th bit
+                                        // const INV2: u128 = 0x1;                // 2**256 // lambda - 128th bit
```

Comment:
Seems this line needs to be reverted manually

Reply:
probably just remove it since it's commented out code

Reply:
Sure.

For the record, INV2 is the highest bit of INV, which was not used in the calculation below and thus commented out.